Trappist/src/Promact.Trappist.Test/Tests/TestsRepositoryTest.cs

```diff
@@ -93,11 +93,11 @@ public TestsRepositoryTest(Bootstrap bootstrap) : base(bootstrap)
             var test = CreateTest("AOT 669");
             await _testRepository.CreateTestAsync(test);
             test.TestName = "IIT BANGALORE";
-            test.BrowserTolerance = 2;
+            test.BrowserTolerance = DomainModel.Enum.BrowserTolerance.High;
             await _testRepository.UpdateTestByIdAsync(test);
             var TestName = "IIT BANGALORE";
             Assert.True(_trappistDbContext.Test.Count(x => x.TestName == TestName) == 1);
-            Assert.True(_trappistDbContext.Test.Count(x => x.BrowserTolerance == 2) == 1);
+            Assert.True(_trappistDbContext.Test.Count(x => x.BrowserTolerance == DomainModel.Enum.BrowserTolerance.High) == 1);
```

Comment:
the browser tolerance values you set at the client side, need to do at server side also.